sharktank/tests/models/clip/clip_test.py

```diff
@@ -96,6 +97,11 @@ def testSmokeExportLargeF32FromHuggingFace(self):
             huggingface_repo_id, output_path
         )
 
+    def testSmokeExportToyIreeTestData(self):
+        from sharktank.models.clip.export_toy_text_model_iree_test_data import main
+
+        main([f"--output-path-prefix={self.path_prefix}clip_toy_text_model"])
```

Comment:
Use pythons path library to operate on this instead of string manipulations.

Reply:
Done. I changed it to use a directory as a prefix.